templates/header/package_navigation.html

```diff
@@ -21,7 +22,7 @@ <h1 id="crate-title">
                     {{ title }}
                 {%- else -%}
                     {{ metadata.name }} {{ metadata.version }}
-                    <i class="fa fa-copy fa-1" id="clipboard" aria-label="Copy crate name and version information"></i>
+                    {{ "copy" | far(extra='id="clipboard" aria-label="Copy crate name and version information"') | safe }}
```

Comment:
You forgot to remove the `| safe` here. Also, this needs to have `aria-hidden="false"`, since we want screen readers to see this icon (since we added a `aria-label` attribute).